web/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
```

Comment:
If we are adding a linter, we should probably also incorporate it in the CI.

Reply:
The linter was present and is also already used in the ci. This change added a comma so the "serve:watch" command could be added.

Reply:
Oops, my bad :facepalm: